src/main/java/com/commercetools/sync/commons/exceptions/InvalidProductTypeException.java

```diff
@@ -0,0 +1,40 @@
+package com.commercetools.sync.commons.exceptions;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+public class InvalidProductTypeException extends Exception {
```

Comment:
all the methods in this class are similar to the other exception class `ReferenceReplacementException `, maybe use a wrapper to not the duplicate code.

Reply:
hmm, let me see what I can do there.

Reply:
be4171f